src/compiler/hxb/hxbReader.ml

```diff
@@ -152,6 +152,7 @@ class hxb_reader
 	(timers_enabled : bool)
 = object(self)
 	val mutable api = Obj.magic ""
+	val mutable read_all = true
```

Comment:
This should probably have a more specific name.

Reply:
Agreed, though I don't have much inspiration at the moment..

Reply:
This https://github.com/HaxeFoundation/haxe/pull/11659/commits/2f4793a55058c94eaddc3f4514d9c9cc1475722f sounds better to me